### PR TITLE
Subaru DBC update

### PR DESCRIPTION
**Pull request by martinl**
Added new messages and signals, mostly related to testing and developing Openpilot longitudinal control on global platform (Crosstrek 2018):
- Brake_Status (Brake pedal and ES_Brake status feedback for Eyesight)
- Cruise_Status (second message with CruiseControl status info)
- AC_State (A/C state)

New and updated signals:
- Throttle Throttle_Combo
- Transmission RPM
- Brake Pedal Brake_Lights
- ES_Brake Cruise_*
- ES_Distance Cruise_*
- ES_Status Cruise_*
- Dashlights Counter (both crosstrek and global dbc)
- BodyInfo Brake
- ES_DashStatus Conventional_Cruise
- ES_DashStatus Cruise_Soft_Disable
- Steering_Torque Steer_Warning
- Brake_Pedal Speed
- ES_LKAS_State LKAS_*

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Can you remove all the NEW_MSG and NEW_SIGNAL definitions or rename them if we know what they are?

**Comment by martinl** [comment]
> Can you remove all the NEW_MSG and NEW_SIGNAL definitions or rename them if we know what they are?

I removed unknown and updated known messages and signals

**Comment by martinl** [comment]
Also, this small test script has been useful for catching some dbc fields overlap and syntax errors
```
import sys
import cantools

dbc_file = sys.argv[1]

try:
  dbc = cantools.database.load_file(dbc_file)
  print("%s OK" % (dbc_file))
except Exception as e:
  print("%s Failed" % (dbc_file))
  print(e)
```

File: subaru_crosstrek_2018.dbc [hunk 1]
```diff
@@ -37,54 +37,52 @@ BU_: XXX X
...
- SG_ Gear : 24|8@1+ (1,0) [0|255] "" XXX
- SG_ NEW_SIGNAL_4 : 38|3@0+ (1,0) [0|255] "" XXX
- SG_ NEW_SIGNAL_3 : 40|16@1+ (1,0) [0|255] "" XXX
+ SG_ CVT_Gear : 24|8@1+ (1,0) [0|127] "" XXX
 
-BO_ 316 NEW_MSG_3: 8 XXX
+BO_ 316 Brake_Status: 8 XXX
+ SG_ CHECKSUM : 0|8@1+ (1,0) [0|1] "" XXX
+ SG_ Counter : 8|4@1+ (1,0) [0|1] "" XXX
+ SG_ Signal1 : 12|46@1+ (1,0) [0|1] "" XXX
```
Comment (adeebshihadeh) [inline subaru_crosstrek_2018.dbc:68]: What are all these `SignalX` definitions for?

Comment (martinl) [reply subaru_crosstrek_2018.dbc:68]: They are filler signals for messages which are copied in carstate and rewritten in subarucan, eg https://github.com/martinl/openpilot/blob/feature-subaru-long/selfdrive/car/subaru/subarucan.py

**Pull request merged**